src/intrprtr.c

```diff
@@ -3921,8 +3928,12 @@ void            IntrInfoEnd( UInt narg )
      Obj args;    /* gathers up the arguments to be printed */
 
     /* ignore or code                                                      */
-    SKIP_IF_RETURNING();
-    if ( STATE(IntrIgnoring)  > 1 ) { STATE(IntrIgnoring)--; return; }
+     INTERPRETER_PROFILE_HOOK(1);
+     SKIP_IF_RETURNING_NO_PROFILE_HOOK();
+
+     if (STATE(IntrIgnoring) > 1) {
+         STATE(IntrIgnoring)--;
+         return; }
```

Comment:
I'd either keep this on one line, as it was; or else fully clang-format it (and get that brace on its own line)

Reply:
Weirdly, this was clang-formatted. However, clang-format seems to have not reached a fixed point, and re-clang-formatting changes it again.